shell/platform/android/io/flutter/view/FlutterView.java

```diff
@@ -693,6 +710,23 @@ private void updateSemantics(ByteBuffer buffer, String[] strings) {
         }
     }
 
+    // Called by native to notify first Flutter frame rendered.
+    private void onFirstFrame() {
+        if (mFirstFrameListeners.isEmpty()) {
+            return;
+        }
+        // Call the listeners on the UI thread.
```

Comment:
We're a bit inconsistent about this, but I'd suggest calling this the "platform thread" or the "Android UI thread".  Elsewhere in the engine the "UI thread" refers to the thread running Flutter Dart UI code.